### ISSUE TITLE
Where do you get trainval.txt?

### ISSUE THREAD

**Issue by Frostinassiky** [ChaoPei/faster-rcnn-pedestrian-detection#2]
Hello,

When I run 
`bash ./experiments/scripts/caltech_end2end.sh 0 ResNet50 caltech_reasonable`, 
I get the following output
`Path does not exist: data/caltech-pedestrian-dataset-converter/data/trainval.txt`

**caltech-pedestrian-dataset-converter** only generate image and annotations. Where do you get _trainval.txt_?

**Comment by Frostinassiky** [comment]
Nope. I failed to fix the issue. I am using detectron framework now. It is more convenient.

**Comment by Sino0904** [comment]
Hi @Frostinassiky  @tranquoclongt1 !
So, you can create trainval.txt at the given path by yourself. Inside the the file, you need to mention which set number you want for training. I did that and its working for me.

**Comment by ChaoPei** [comment]
@Frostinassiky @tranquoclongt1 

Hi, sorry for late reply, the trainval.txt is just like the train.txt and text.txt that you can create by yourself.@Sino0904 is right
